helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -106,6 +106,18 @@ splunkPlatform:
     #   requests_per_second is the average number of requests per seconds.
     queueSize: 5000
 
+    # When enabled, it uses file_storage extension to persist the queue data.
+    # This can be used as an alternative redundancy mechanism for data being exported.
+    # NOTE: The File Storage extension will persist state to the node's local file system.
+    # While using the persistent queue it is advised to increase memory limit for agent (agent.resources.limits.memory)
+    # to 1Gi. Persistent queue is currently supported only for logs and metrics of the agent.
```

Comment:
Where is that memory recommendation coming from?

Reply:
From sizing guide: https://github.com/signalfx/splunk-otel-collector-chart/blob/main/docs/troubleshooting.md#sizing-splunk-otel-collector-containers